core/include/kernel_init.h

```diff
@@ -25,6 +25,27 @@
 extern "C" {
 #endif
 
+/**
+ * @name    Boot Message configuration
```

Comment:
For this to show up in the compile configuration section it should be something like:
```C
/**
 * @defgroup core_init_config Core initialization compile configuration
 * @ingroup config
```

Reply:
Ok!